webapp/src/services/service.ts

```diff
@@ -0,0 +1,22 @@
+import client from "./client";
+
+export interface Services {
+  id: number;
+  name: string;
+  serviceGroup?: string;
```

Comment:
```suggestion
  serviceGroup: string | null;
```

Reply:
A question mark in C# means a type is nullable but it still cannot be `undefined`.